---
description: 了解详细信息： System.identitymodel. IdentityModelAsyncCallbackThrewException
title: System.IdentityModel.IdentityModelAsyncCallbackThrewException
ms.date: 03/30/2017
ms.assetid: e4fc924f-3b9e-44f5-97e4-b3a4bb855d89
ms.openlocfilehash: 304cf448b71c6a0aa901a09273b81b686020d0f2
ms.sourcegitcommit: ddf7edb67715a5b9a45e3dd44536dabc153c1de0
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 02/06/2021
ms.locfileid: "99635280"
---
# <a name="systemidentitymodelidentitymodelasynccallbackthrewexception"></a><span data-ttu-id="04a1d-103">System.IdentityModel.IdentityModelAsyncCallbackThrewException</span><span class="sxs-lookup"><span data-stu-id="04a1d-103">System.IdentityModel.IdentityModelAsyncCallbackThrewException</span></span>

<span data-ttu-id="04a1d-104">System.IdentityModel.IdentityModelAsyncCallbackThrewException</span><span class="sxs-lookup"><span data-stu-id="04a1d-104">System.IdentityModel.IdentityModelAsyncCallbackThrewException</span></span>  
  
## <a name="description"></a><span data-ttu-id="04a1d-105">说明</span><span class="sxs-lookup"><span data-stu-id="04a1d-105">Description</span></span>  

 <span data-ttu-id="04a1d-106">异步回调引发异常。</span><span class="sxs-lookup"><span data-stu-id="04a1d-106">An asynchronous callback threw an exception.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="04a1d-107">请参阅</span><span class="sxs-lookup"><span data-stu-id="04a1d-107">See also</span></span>

- [<span data-ttu-id="04a1d-108">跟踪</span><span class="sxs-lookup"><span data-stu-id="04a1d-108">Tracing</span></span>](index.md)
- [<span data-ttu-id="04a1d-109">使用跟踪来排除应用程序故障</span><span class="sxs-lookup"><span data-stu-id="04a1d-109">Using Tracing to Troubleshoot Your Application</span></span>](using-tracing-to-troubleshoot-your-application.md)
- [<span data-ttu-id="04a1d-110">管理和诊断</span><span class="sxs-lookup"><span data-stu-id="04a1d-110">Administration and Diagnostics</span></span>](../index.md)
